canonical-web-and-design/ubuntu.com
# fix s390x resource row

## Done

fix s390x resource row

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/server/thank-you-s390x
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the resource row looks modern
